### PR TITLE
Replace pydot with graphviz

### PR DESCRIPTION
**Pull request by eduardo-rodrigues**
Indeed `pydot` isn't really maintained since a while. Better be on the safe side.

Closes https://github.com/scikit-hep/decaylanguage/issues/105.

### CODE REVIEW

**Comment by eduardo-rodrigues** [comment]
@henryiii, I don't understand how Sourcery is working. It is failing my PR and making a close in 128. Why is that? I would have expected to get suggested changes here, full stop. Hmm.

In any case the PR is ready for review :).

File: setup.py [hunk 1]
```diff
@@ -70,7 +69,7 @@ def get_version():
     install_requires=INSTALL_REQUIRES,
     tests_require=extras["test"],
     extras_require=extras,
-    keywords=["HEP", "particle", "decay", "representation"],
+    keywords=["HEP", "particle", "decay", "chain" "representation"],
```
Comment (henryiii) [inline setup.py:72]:
```suggestion
    keywords=["HEP", "particle", "decay", "chain", "representation"],
```

This is currently:
```python
    keywords=["HEP", "particle", "decay", "chainrepresentation"],
```

Which I don't think is what you meant.

**Comment by henryiii** [comment]
In the near future I'll adapt to follow the scikit-hep dev guidelines here and in Particle.

File: setup.py (hunk 1 above)
Comment (henryiii) [reply setup.py:72]: I think our flake8 (at least bugbear) testing would catch this.

Comment (eduardo-rodrigues) [reply setup.py]: My eyes did not see the missing comma. Yes, that would be found, I guess too.

**Comment by eduardo-rodrigues** [comment]
> In the near future I'll adapt to follow the scikit-hep dev guidelines here and in Particle.

Indeed this package hasn't yet been updated for all guidelines, and it would be great :-).

**Comment by eduardo-rodrigues** [comment]
Any other comment on this PR, @henryiii? Else I merge. We can try and have the package reorganised (e.g. code under src/) according to guidelines for the next minor release, then, as you suggested above.

**Comment by henryiii** [comment]
Sounds good, I can try to help with some of the modernization after this PR is merged.

**Review by henryiii — APPROVED** [review]

**Comment by eduardo-rodrigues** [comment]
> Sounds good, I can try to help with some of the modernization after this PR is merged.

BTW, on that - have you done the modernisation of various packages "by hand" or ...? I can start off if you like.

**Comment by henryiii** [comment]
Setup.py -> setup.cfg I have mostly automated. https://github.com/henryiii/setuptools-modernize

Other than that one, mostly manual.

**Comment by eduardo-rodrigues** [comment]
OK, thanks!

**Pull request merged**